web/src/components/NewsListItem.tsx

```diff
@@ -61,7 +57,12 @@ const NewsListItem = ({ title, content, timestamp, formatter, t, type, link }: P
       <Link to={link}>
         <Description>
           <Title>{title}</Title>
-          <Body>{textTruncator(content, NUM_OF_CHARS_ALLOWED)}</Body>
+          <RemoteContent
```

Comment:
Actually I think we should not render the text as actual HTML here, otherwise formatting, links and so on will be applied. E.g. links do not make any sense as clicking on it just opens the news details page and not the link.

Reply:
@ klinzo 4) sounds like a nice and simple solution but I don't undestand how to apply it. The following does not work: 
```
const Body = styled.p`
  font-size: 16px;
  line-height: 1.38;
  white-space: pre-line;
`
```

Reply:
Fixed. I took solution 4)